resolving/npm-resolver/test/index.ts

```diff
@@ -1018,7 +1018,7 @@ test('resolve from local directory when it matches the latest version of the pac
     ]),
   })
 
-  expect(resolveResult!.resolvedVia).toBe('local-filesystem')
+  expect(resolveResult!.resolvedVia).toBe('workspace')
```

Comment:
I didn't think we needed a new test to check for `workspace` is returned in `resolvedVia` since the existing tests cover it. Let me know if we want a new test dedicated for this though.